app/form_models/jobseekers/search_form.rb

```diff
@@ -45,11 +45,14 @@ def strip_trailing_whitespaces_from_params(params)
   end
 
   def set_facet_options
-    @job_role_options = Vacancy.job_roles.keys.map { |option| [option, I18n.t("helpers.label.publishers_job_listing_job_details_form.job_roles_options.#{option}")] }
+    @job_role_options = Vacancy.job_roles.except(:nqt_suitable).keys.map { |option| [option, I18n.t("helpers.label.publishers_job_listing_job_details_form.job_roles_options.#{option}")] }
     @phase_options = [%w[primary Primary], %w[middle Middle], %w[secondary Secondary], %w[16-19 16-19]]
+    @suitable_nqts_options = Vacancy.job_roles.slice(:nqt_suitable).keys.map { |option| [option, I18n.t("helpers.label.publishers_job_listing_job_details_form.job_roles_options.#{option}")] }
```

Comment:
We're only ever going to have one option here, right? So we could just put `I18n.t("helpers.label.publishers_job_listing_job_details_form.job_roles_options.nwt_suitable")`
?

Reply:
when we changed this line end of last week it did actually stop working @david-mears-dfe